src/rexsdesktop/modal/ModalNuevoProyecto.java

```diff
@@ -64,7 +64,7 @@ public ModalNuevoProyecto() {
     public void cargarProyectos() {
         try {
             General.getEdicion();
-            db.NumProyectos(CurrentUser.edicionExpotecnica);
+            db.NumProyectos(CurrentUser.getEdicionExpotecnica());
```

Comment:
You are accessing to the DB from the view, this should happen through the controller